src/info.jl

```diff
@@ -51,3 +33,49 @@ By default, returns `nothing` as info.
 function update_info(up::Updater, b, a, o)
     return update(up, b, a, o), nothing
 end
+
+"""
+    add_infonode(ddn::DDNStructure)
+
+Create a new DDNStructure object with a new node labeled :info with parents :s and :a
```

Comment:
I think it would be valuable to have the example from the conversation in here

Reply:
:+1: it is in the docstring now